src/SFA.DAS.AANHub.Database/Tables/EventGuest.sql

```diff
@@ -0,0 +1,12 @@
+﻿CREATE TABLE [dbo].[EventGuest]
+(
+    [Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY DEFAULT NEWID(), 
+    [CalendarEventId] UNIQUEIDENTIFIER NOT NULL FOREIGN KEY REFERENCES [CalendarEvent]([Id]),  

```

Comment:
That is not the plan. Guests are Guests and members can attend.
Latest discussions were that If a memebr is to be a guest then this will be added as a Guest.
Something for the Ui to resolve but not the table.